src/scripts/main.js

```diff
@@ -1,3 +1,13 @@
 'use strict';
 
-// write your code here
+const population = [...document.getElementsByClassName('population')];
+const totalPopulationClass = document.querySelector('.total-population');
+const averagePopulationClass = document.querySelector('.average-population');
+
+const total = population.reduce((sum, currentPopulation) =>
+  sum + +currentPopulation.innerHTML.split(',').join(''), 0);
```

Comment:
ты просто оборвал строку и на этом все, читаемость не очень
```suggestion
const total = population.reduce((sum, currentPopulation) => (
  sum + +currentPopulation.innerHTML.split(',').join('')
), 0);
```